client/src/pages/SignUp/index.tsx

```diff
@@ -0,0 +1,113 @@
+import './style.css';
+import Avatar from '@mui/material/Avatar';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import Link from '@mui/material/Link';
+import Grid from '@mui/material/Grid';
+import Box from '@mui/material/Box';
+import PermIdentityOutlinedIcon from '@mui/icons-material/PermIdentityOutlined';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import { faGoogle, faLinkedin } from "@fortawesome/free-brands-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```

Comment:
lol is this really "fortawesome" for the path 🤣 I see that it's working fine, but that's so odd

Reply:
yeah i was thinking the same thing. I thought I have having a stroke or something when i read that it was fortawesome lol